src/_gettsim/transfers/unterhaltsvors.py

```diff
@@ -113,3 +125,28 @@ def unterhaltsvorschuss_eink_m_tu(  # noqa: PLR0913
     )
 
     return out
+
+
+def kind_unterh_netto_m(
```

Comment:
I Just realised that this should go to a separate module `unterhalt.py` (no need to worry about inconsistency `unterh_` vs. `unterhalt` in this PR).

Reply:
Alright, I will change that. Will we also need separate tests for this function? I think the missing tests are the reason for the decreased coverage

Reply:
Yeah, would be good, trivial as they may be...